README.md

```diff
@@ -30,6 +30,10 @@ The fields in the table below can be used in these parts of STAC documents:
 | forecast:reference_datetime  | string | **REQUIRED.** The *reference* datetime: i.e. predictions for times after this point occur in the future. Predictions prior to this time represent 'hindcasts', predicting states that have already occurred. This must be in UTC. It is formatted according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). |
 | forecast:horizon             | string | The time between the reference datetime and the forecast datetime. Formatted as [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations), e.g. `PT6H` for a 6-hour forecast. |
 | forecast:duration            | string | If the forecast is not only for a specific instance in time but instead is for a certain period, you can specify the length here. Formatted as [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations), e.g. `PT3H` for a 3-hour accumulation. If not given, assumes that the forecast is for an instance in time as if this was set to `PT0S` (0 seconds). |
+| forecast:param               | string | Name of the model parameter that corresponds to the data, e.g. `T` (temperature), `P` (pressure), `U`/`V`/`W` (windspeed in three directions).  **TODO: should this be either string or list of strings to allow for situations where several parameters are combined in a file?**|
+| forecast:mode                | string | Denotes whether the data corresponds to the control run or perturbed runs. Allowed values are `ctrl` or `perturb`. |
+| forecast:sample_nr           | integer | Specifies the sample number of perturbed runs, e.g. `1`. **TODO: this could be omitted for the moment and added when there's a concrete use case** |
```

Comment:
What about "member"? Or realization as in the netcdf conventions https://cfconventions.org/cf-conventions/cf-conventions.html?utm_source=perplexity

Reply:
I think it would be nice to follow existing conventions such as CF, indeed.

Reply:
@aaronspring @m-mohr thanks for the review and the suggestion. I'll rename the attribute to `forecast:member`.